scripts/deploy_heroku.sh

```diff
@@ -32,4 +32,14 @@ docker push $HEROKU_IMAGE
 # TODO(hoatle): hide the secret output on gitlab-ci console
 echo "updating heroku configs";
 
-heroku config:set SECRET_KEY_BASE=$SECRET_KEY_BASE -a $HEROKU_APP_NAME &> /dev/null # hide secret output
+heroku config:set SECRET_KEY_BASE=$SECRET_KEY_BASE \
+  DATABASE_HOST=$HEROKU_DATABASE_HOST \
+  DATABASE_PRODUCTION=$HEROKU_DATABASE \
+  DATABASE_USER=$HEROKU_DATABASE_USER \
```

Comment:
Yes, sorry about that. I didnt test my lastest commit after your first review. I will test more